README.md

```diff
@@ -1,3 +1,27 @@
 # Secrets of Sosaria
 
 For instructions, see [Manual.pdf](Manual.pdf).
+
+
```

Comment:
Would you mind adding a small section for running locally directly on Windows?  I think this would avoid some of the confusion around if you need to run docker to run SoS.

Reply:
Sorry, I have no Windows PC around, only MacOS and Linux, so I can add only that part. I'd be grateful for everyone who can contribute though!